GSA-TTS/FAC-transition-site
# Restructure Images Folder

# Restructure Images Folder

Issue: https://github.com/GSA-TTS/fac-team/issues/32

## Future Image Management

Should be a little more simple! I've axed the date stamped folders for use in the collections files. They're a little tough to manage because there are lots of duplicated files.

1. Images should go in `assets/img/`, under whichever folder is most relevant. 
2. Images that would be included in several pages, such as screenshots of the home and login screens, go in the `assets/img/shared/` folder
    * Updating a shared image in-place (new image, same name and folder) will update it in all the relevant spots. For example, if Login ever changes their look, we can update just the one screenshot in `shared/`
3. When changing a file name or location, it should be updated in the relevant collections file.
    * These files have been updated to access `img` first, rather than their subfolder. So they can get to the shared folder.
    * `walkthrough_01.png` is now either `shared/home_annotated.png` or something like `walkthrough/the_first_step.png`

## Changes:

1. Remove date stamped folders
2. Collections use more full path names, to allow accessing other folders
3. A new `shared` folder to help prevent duplicating images

As a bonus:
1. Removed extra whitespace on some collections
2. Removed an unused image (`tabular_thing.png`)
3. Removed an unused component (`hero.png`)
    * USWDS has their own version embedded in [landing-page](https://designsystem.digital.gov/templates/landing-page/) that we can reference if we ever need to

## How to test: 
1. Switch to this branch and run normally
2. Verify that the affected pages display the correct images:
    * [Tribal](http://localhost:8080/tribal/)
    * [Submission guide](http://localhost:8080/audit-resources/how-to/)
    * [Access management](http://localhost:8080/audit-resources/user-access/)
